tools/flaky_test_monitor/testdata/summary2/test4-multi-failures/expected-output/test4-multi-failures.json

```diff
@@ -0,0 +1,148 @@
+{
+  "tests": 
+    {
+      "github.com/onflow/flow-go/crypto/hash/TestSanitySha3_256":
+      {
+        "test": "TestSanitySha3_256",
+        "package": "github.com/onflow/flow-go/crypto/hash",
+        "runs": 25,
+        "passed": 24,
+        "failed": 1,
+        "skipped": 0,
+        "no_result": 0,
+        "failure_rate": 0.04,
+        "average_duration": 0,
+        "durations": [
+          0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
```

Comment:
conceptually, zero duration doesn't really make sense to me, what exactly does these mean?

Reply:
Many test run durations are reported as 0 by `go test` so we don't have any more precision into duration values unless we built our own duration timer.

We need to keep track of how long each test run takes so we can determine what are the longest running tests and calculate average duration for a test.